### PR TITLE
Categories added

### LINKED ISSUE #5: Categorize topics by catergory
For eg: A catergory, Games in topics

If clicked, buttons for: 
Minecraft
Roblox
Football

### PR DESCRIPTION
**Pull request by KrishJ4856**
### As Per The Issue, I have added the different Categories according to my understanding of the different quizes

Fixes #5 

## Type of change
New feature (non-breaking change which adds functionality)

# Screenshots

Before: 

[](https://postimg.cc/bsrtgszF)

[](https://postimg.cc/1f8862Hk)

After:

[](https://postimg.cc/0zHKhxN2)

### CODE REVIEW

File: topics/index.html [hunk 1]
```diff
@@ -4,9 +4,14 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz Topics</title>
+    <title>Trying</title>
```
Comment (KendallDoesCoding) [inline topics/index.html:7]: Why did you change it to Trying? I require it to be Quiz Topics, please revert it back.

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
Please delete `.vscode/settings.json`

**Comment by KrishJ4856** [comment]
Hey @KendallDoesCoding  I am extremely sorry for the delay. I have been making few mistakes with commits and PRs. 
I have closed this PR and raised a new one because I could not find solutions to the mistakes I made. I have made the necessary changes, please review it. 
Thanks and once again sorry for the inconvenience.
